R/zzz.R

```diff
@@ -7,22 +7,31 @@
 
 # nocov start
 register_mlr3 = function(libname, pkgname) {
+
   # get mlr_learners dictionary from the mlr3 namespace
   x = utils::getFromNamespace("mlr_learners", ns = "mlr3")
 
   # add the learner to the dictionary
+  x$add("classif.AdaBoostM1", LearnerClassifAdaBoostM1)
+  x$add("classif.IBk", LearnerClassifIBk)
+  x$add("regr.IBk", LearnerRegrIBk)
   x$add("classif.JRip", LearnerClassifJRip)
   x$add("classif.J48", LearnerClassifJ48)
+  x$add("classif.LMT", LearnerClassifLMT)
   x$add("regr.M5Rules", LearnerRegrM5Rules)
+  x$add("classif.OneR", LearnerClassifOneR)
+  x$add("classif.PART", LearnerClassifPART)
 }
 
-.onLoad = function(libname, pkgname) { # nolint
+.onLoad = function(libname, pkgname) {
+  # nolint
```

Comment:
this won't work - update {styler} installation to fix the unwanted linebreaking issue